server/schedulers/hot_region.go

```diff
@@ -288,12 +297,17 @@ func summaryStoresLoad(
 			}
 		}
 
+		//calculate the storeWeight
+		storeWeight := calcStoreWeight(
+			cluster.GetStore(id).GetRegionWeight(),
```

Comment:
This weight may need to be separated from the cluster weight, because this is related to load, and the previous weight is more related to size. you can wait for https://github.com/pingcap/pd/pull/2239.

Reply:
OK.I will do more test and wait for #2239.